cli/bin/internal_command_update_prompt.dart

```diff
@@ -27,16 +27,8 @@ class InternalCommandUpdatePrompt extends InternalCommand {
         return false;
       }
     }
-    bool answer = context.commandLineContinueQuestion(
-        "Stax will update after executing your command.");
-    if (answer) {
-      context.printToConsole(
-          "Thanks for supporting stax and updating it to latest version!");
-    } else {
-      context.printToConsole(
-          "Ok, will ask again in ${silenceDuration.inDays} day(s).");
-    }
-    return answer;
+
```

Comment:
Let's revert changes to this file, and you will be good to go.

Reply:
revert done